unittests/InstructionCountCI/FlagM/Primary_32Bit.json

```diff
@@ -247,67 +247,53 @@
       ]
     },
     "pusha": {
-      "ExpectedInstructionCount": 10,
+      "ExpectedInstructionCount": 6,
       "Comment": "0x60",
       "ExpectedArm64ASM": [
-        "mov w20, w8",
-        "str w4, [x20, #-4]!",
-        "str w7, [x20, #-4]!",
-        "str w5, [x20, #-4]!",
-        "str w6, [x20, #-4]!",
-        "str w8, [x20, #-4]!",
-        "str w9, [x20, #-4]!",
-        "str w10, [x20, #-4]!",
-        "mov w8, w20",
-        "str w11, [x8, #-4]!"
+        "mov x20, x8",
```

Comment:
This move is actually unsafe and the guest stack will get corrupted on asynchronous signal. Need to ensure not to allow moves like this.

Reply:
This isn't a regression from the previous code generated for the same instruction, no?

Reply:
I'm redoing the pusha stuff anyway